app/views/shared/_navbar.html.erb

```diff
@@ -1,11 +1,10 @@
 <div class="navbar navbar-expand-sm navbar-light navbar-lewagon white fixed-top">
   <div>
     <%= link_to fabrics_path, class: "navbar-brand" do %>
-      <%= image_tag "logo.png" %> 
+      <%= image_tag "logo.png" %><small class="ml-4">Da Pluma à Pele</small>
```

Comment:
não sei se funciona isso, pois tem js na Home agora (por isso tinha mudado)

Reply:
entao, esse texto ta dentro do link to.

nao entendi o que acha que pode dar erro?